frontend/app/components/address.tsx

```diff
@@ -19,15 +20,27 @@ function formatAddress(address: string, city: string, country: string, provinceS
     `${city}${provinceState ? ` ${provinceState}` : ''}${postalZipCode ? `  ${postalZipCode}` : ''}`,
     `${isNotCanadianAddress ? country : ''}`];
 
+  // prettier-ignore
+  const linesAlt = [`${address}`,
+  `${city}${provinceState ? ` ${provinceState}` : ''}`,
+  `${postalZipCode ? `  ${postalZipCode}` : ''}`,
```

Comment:
There are two spaces (`  `) added here, but then they are trimmed a few lines down.

Same for the `lines` value just above this, btw.

Reply:
FYI the `lines` value above will only get trimmed with the two spaces if `city` is left empty (it's a required field though). 

The two spaces is because of the Canada Post guidelines for addresses: 
> Separate the postal code from the province symbol by 2 spaces.

See https://www.canadapost-postescanada.ca/cpc/en/support/kb/business/address-accuracy/addressing-mail-accurately

I feel like we should stick with a consistent way of formatting the address. I know this `linesAlt` is intended to match the current Figma mock-ups. We should stick with that way or the Canada Post format.

Reply:
Removed the extra space as mentioned, the address styling for `lines` is also used in `address-accuracy.tsx` and it might change the styling from the Figma, do we still want to replace this with `linesAlt` or use both?

Reply:
@yannismassyl will ask the designers about the address format. I am okay with leaving `linesAlt` for now and removing later.